modules/cluster-and-ha-management/pages/ha-overview.adoc

```diff
@@ -22,12 +22,18 @@ TigerGraph HA provides continuous operation of some but not all services.
 Please note the following exceptions and consider your options for taking additional steps to maintain continuous operation or to restore service as quickly as possible.
 
 .If an HA system is operating with a failed node, unless the node is recovered or the system is reconfigured to xref:tigergraph-server:cluster-and-ha-management:remove-failed-node.adoc[exclude that node], the following services are limited or unavailable:
-* A file-based data load cannot be loaded for the failed node.
-* A data partition slated for a kafka-based data load cannot be loaded.
-* New queries cannot be installed.
-* Schema changes are not allowed.
-* Backup and export operations are not available and will be rejected.
-* If the primary node is offline, access to Graph Studio is interrupted, but resumes once the primary node is back online.
+
+* A data partition slated for a connector-based loading, such as, s3 files or via kafka, *cannot* be loaded.
+
+* New queries *cannot* be installed.
++
+NOTE: However, new interpreted and any existing queries can still be executed.
+
+* Schema changes are *not* allowed.
+
+* Backup and export operations are *not available* and will be rejected.
+
+* If the primary node is offline, access to Graph Studio is *interrupted*, but resumes once the primary node is back online.
 
 NOTE: As a workaround, if the failed node cannot be recovered (e.g. hardware issue), full operation can be restored temporarily by the xref:tigergraph-server:cluster-and-ha-management:remove-failed-node.adoc[removal of the failing nodes].
```

Comment:
or replaced by a good one

Reply:
Thank you Songting. I have updated this the paragraph above this one, for a better flow, when reading it.

"If an HA system is operating with a failed node, unless the node is recovered, xref:tigergraph-server:cluster-and-ha-management:how_to-replace-a-node-in-a-cluster.adoc[replaced], or the system is reconfigured to xref:tigergraph-server:cluster-and-ha-management:remove-failed-node.adoc[exclude that node], the following services are limited or unavailable:"

Adding the replacing cluster instructions.